content/batch/artifact.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Configure Artifact Repository"
+date: 2018-11-18T00:00:00-05:00
+weight: 50
+draft: false
+---
+
+### Configure Artifact Repository
+
+Argo uses an artifact repository to pass data between jobs in a workflow, known as artifacts. Amazon S3 can be used as an artifact repository.
+
+Let's create a S3 bucket using the AWS Service Operator from earlier.
+
+```bash
+ACCOUNT_ID=$(aws sts get-caller-identity --output text --query Account)
+
+cat <<EoF > ~/environment/batch-artifact-repository.yaml
+---
+apiVersion: operator.aws/v1alpha1
+kind: S3Bucket
+metadata:
+  name: batch-artifact-repository-${ACCOUNT_ID}
+spec:
+  versioning: false
+EoF
+
+kubectl apply -f ~/environment/batch-artifact-repository.yaml
```

Comment:
I got error here, 
dalbhanj:~/environment $ kubectl apply -f ~/environment/batch-artifact-repository.yaml
error: unable to recognize "/home/ec2-user/environment/batch-artifact-repository.yaml": no matches for kind "S3Bucket" in version "operator.aws/v1alpha1"

Reply:
@christopherhein Can you help here with getting the operator working? I had trouble as well. @dalbhanj See the "alternate" instructions to create the S3 bucket using the AWS CLI.

Reply:
Changed to use AWS CLI.